salt/metalk8s/repo/offline.sls

```diff
@@ -4,10 +4,14 @@
 {%- set repo_host = pillar.metalk8s.endpoints['repositories'].ip %}
 {%- set repo_port = pillar.metalk8s.endpoints['repositories'].ports.http %}
 
+Set osmajorrelease in yum vars:
+  file.managed:
+    - name: /etc/yum/vars/osmajorrelease
```

Comment:
I think we should prefix with `metalk8s` even if it's major release as it's something used by us

Reply:
done